### ISSUE TITLE
Parameters of taking a picture of the test form

### ISSUE THREAD

**Issue by ZahraHosseini-04** [sadafnazari97/Persian_handwriting_recognition#12]
Hello,
In order to get a better output from the program, what features should be observed in taking pictures?
Thank you for your guidance.

**Comment by sadafnazari** [comment]
Hi,
If you want to get better accuracy there are two options:
-First option would be to add more data. The instructions are already given in README. Note that there should not be an attempt to take data with perfect conditions as the training data should be as general as possible with various handwritten styles for each letter and number. 
-Second option would be the hyper parameter tuning. I have not had time to train the model with different hyper parameters such as the model structure itself or other factors such as number of epochs.

Br,
Sadaf Nazari

**Comment by ZahraHosseini-04** [comment]
Hello
Yes, I understand.
Thank you for your explanation.
